bbb-common-web/src/main/java/org/bigbluebutton/api/model/validator/ContentTypeValidator.java

```diff
@@ -31,7 +31,12 @@ public boolean isValid(Request request, ConstraintValidatorContext context) {
         if (requestBodyPresent) {
             if (contentType == null || contentTypeHeader == null) return false;
             else {
-                return request.getSupportedContentTypes().contains(contentType);
+                String[] contentTypeParts = contentType.split(";");
+                for (Object o: request.getSupportedContentTypes()) {
+                    String supportedContentType = (String) o;
+                    if (contentTypeParts[0].equals(supportedContentType)) return true;
```

Comment:
```java
contentTypeParts[0].trim().toLowerCase().equals(supportedContentType.toLowerCase())
```

Should be safer

Reply:
Okay, updated the check to trim whitespace from the content type string and ignore case when comparing the two strings.